File: python101/py10Elágazások/euro.py
# -*- coding: utf-8 -*-

# Euro <-> Ft átváltó program

# Árfolyam beolvasása


# Menü kiírása (1: Euro -> Ft, 2: Ft->Euró)
# Válasz beolvasása
arfolyam = float(input('Hány Ft 1 euro? '))

print('1: Euro -> Ft, 2: Ft-> Euro')
valasz = input('Választás: ')
# Ha a válasz 1
#    Összeg beolvasása EUR-ban
#    Összeg * Árfolyam kiírása egészekre kerekítve
# Egyébként (ha a válasz 2 vagy más)
#    Összeg beolvasása Ft-ban
#    Összeg/Árfolyam kiírása két tizedessel
if valasz == '1':
    osszeg = float(input('Átváltandó összeg (EUR): '))
    print('Az összeg Ft-ban:', round(osszeg*arfolyam))
else:
    osszeg = float(input('Átváltandó összeg (Ft): '))
    print ('Az összeg EUR-ban:', round(osszeg/arfolyam,2))

print("hello world")

File: java101/Alapok/2017.10.01 vezérlés-if,switch/euro/src/euro/Euro.java
package euro;

import java.util.Scanner;

public class Euro {

    public static void main(String[] args) {
        Scanner bill = new Scanner(System.in);
        System.out.print("Hány Ft 1 Euro? ");
        int arfolyam = bill.nextInt();
        int osszeg;
        
        System.out.print("1-Euro->Ft 2-Ft->Euro: ");
        int valasz = bill.nextInt();
        
        switch (valasz) {
            case 1:
                System.out.print("Átváltandó összeg (EUR): ");
                osszeg = bill.nextInt();
                System.out.println(Math.round(osszeg*arfolyam) + " Ft");
                break;
            case 2:
                System.out.print("Átváltandó összeg (Ft): ");
                osszeg = bill.nextInt();
                System.out.println(Math.round(osszeg*arfolyam) + " EUR");
                break;
            default:
                System.out.println("Hibás választás!");
                break;
        }
    }
}
